src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -1867,7 +1867,7 @@ class C
             var active = GetActiveStatements(src1, src2);
 
             edits.VerifyRudeDiagnostics(active,
-                Diagnostic(RudeEditKind.ModifiersUpdate, "const int a = 1", FeaturesResources.const_field));
+                Diagnostic(RudeEditKind.ModifiersUpdate, "a = 1", FeaturesResources.const_field));
```

Comment:
❓ why does this code change from `const int a = 1` to  `a = 1`? is the `const int` no longer important for the test?

Reply:
It's the span of the squiggle. The syntax reference associated with the symbol is different from the syntax used by the previous analysis.